src/taco-cli/cli/utils/cordovaHelper.ts

```diff
@@ -159,6 +159,13 @@ class CordovaHelper {
         return CordovaHelper.toCordovaArgumentsInternal(commandData, platforms);
     }
 
+    public static toCordovaTargetsArguments(commandData: commands.ICommandData, platforms: string[] = null): Cordova.ICordovaRawOptions {
+        // Run, build, emulate, prepare and compile all use the same format at the moment
+        return CordovaHelper.toCordovaArgumentsInternal(commandData, platforms);
```

Comment:
Do we really need separate implementations if they are the same? Is this going to change in the future based on the command?